pages/guides/developing/encrypted-vod.en-US.mdx

```diff
@@ -140,6 +140,10 @@ We're all set to upload the video file.
 When requesting an upload to the Livpeer API, you can simply add an `encryption` field with your encrypted key.
 [Api Reference](https://docs.livepeer.org/reference/api/create-asset-via-url)
 
+<Callout>
+  When uploading an encrypted video, specifying a `playbackPolicy` is required.
```

Comment:
Maybe worth noting that you can still specify it as `public` if that's what you want, but beware that anyone with the `playbackId` or `playbackUrl` would be able to watch it